src/components/ZetkinJourneyInstanceCard.tsx

```diff
@@ -1,38 +1,20 @@
-import { Grid, Typography } from '@material-ui/core';
-
 import ClickableCard from './views/SuggestedViews/ClickableCard';
-import JourneyStatusChip from './journeys/JourneyStatusChip';
-import { ZetkinJourneyInstance } from 'types/zetkin';
+import ZetkinJourneyInstanceItem, {
+  ZetkinJourneyInstanceItemProps,
+} from './ZetkinJourneyInstanceItem';
 
 interface ZetkinJourneyInstanceCardProps {
-  instance: Pick<ZetkinJourneyInstance, 'closed' | 'title' | 'journey' | 'id'>;
+  instance: ZetkinJourneyInstanceItemProps['instance'];
+  orgId: number | string;
 }
 
 const ZetkinJourneyInstanceCard: React.FC<ZetkinJourneyInstanceCardProps> = ({
```

Comment:
Nice to have this component, and the Item extract refactor here. 
Folder structure comment: I think all the `ZetkinJourney*` components should go into their own folder, like e.g. `ZetkinCalendar`. The same goes for `ZetkinPerson*` actually!

Reply:
I like that! I think there is some major refactoring needed to have a good sustainable folder structure for reusable components, and I think we're only now starting to learn what those patterns could be. I think we should do all of that as part of a design system review when we have a UX person on board, i.e. not now.

Do you think that makes sense?